### ISSUE TITLE
Level attribute not used in matching

### ISSUE THREAD

**Issue by pawelpabich** [NLog/NLog#423]
Hi,

My understanding of https://github.com/nlog/NLog/wiki/Configuration-file#rules is that both logger name and log level are used during the matching phase. If not and only the name is used then please disregard this issue. 

I'm trying to disregard NSB debug output.  My expectation was that it is enough to add one more rule to the config file:

```
<logger name="NServiceBus.*" level="Debug"  final=true  />
```

Unfortunately that's not true and all log levels were captured by that rule. To work around this problem I had to basically copy and past all generic rules I had and scope them to NServiceBus.*.

```
  <logger name="NServiceBus.*" level="Debug"  />
    <logger name="NServiceBus.*" minlevel="Info" writeTo="Web"/>
    <logger name="NServiceBus.*" minlevel="Warn" writeTo="Database" final="true"/>
    
    <logger name="*" minlevel="Debug" writeTo="Web"/>
    <logger name="*" minlevel="Warn" writeTo="Database"/>
```

Am I doing something wrong here?

**Comment by 304NotModified** [comment]
The pull request has been merged. Thus I assume the issue has been fixed. If not, then just re-open the issue.

**Comment by pawelpabich** [comment]
Thanks a lot! Are you planning to release a new version of nlog on nuget?

**Comment by ilya-g** [comment]
It will be released in v4.0. @304NotModified, can you tag this issue with v4.0 milestone?
